pkg/daemon/ceph/client/pool.go

```diff
@@ -582,3 +615,24 @@ func GetPoolStatistics(context *clusterd.Context, clusterInfo *ClusterInfo, name
 
 	return &poolStats, nil
 }
+
+func crushRuleExists(crushMap CrushMap, ruleName string) bool {
```

Comment:
`util.NewSet()`?

Reply:
`crushMap` is obtained by running a ceph command and un-marshalling result  into `CrushMap` struct (in `GetCrushMap` method).  

So don't think I can use `util.NewSet()` here.